easybuild/framework/easyblock.py

```diff
@@ -246,6 +260,20 @@ def close_log(self):
         fancylogger.logToFile(self.logfile, enable=False)
 
     #
+    # INIT UTILITIES
+    #
+    def init_dry_run(self):
+        """Initialise easyblock instance for performing a dry run."""
+
+        # replace build/install dir
+        self.builddir = os.path.join(tempfile.gettempdir(), '__ROOT__', self.builddir.lstrip(os.path.sep))
+        dry_run_msg("Using fake build directory: %s" % self.builddir, silent=self.silent)
```

Comment:
We should display this more clearly: keep using a 'fake' build and install dir but displaying them nicely like: `$EASYBUILD_BUILDDIR/...`.